File: 2023/java/src/test/java/org/frieder/aoc/day9/TestDay9A.java
package org.frieder.aoc.day9;

import org.frieder.aoc.day9.a.Solution9A;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay9A {
    @Test
    public void testExample() throws IOException {
        assertEquals(114, Solution9A.getResult("../resources/9/test_a.txt"), 0);
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(1974232246, Solution9A.getResult("../resources/9/input.txt"), 0);
    }
}

File: 2023/php/test/TestDay9A.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day9\a\Solution9A;
use PHPUnit\Framework\TestCase;
use utils\TestLogger;

final class TestDay9A extends TestCase
{

    public function testExample(): void
    {
        $this->assertSame(114, Solution9A::getResult("../resources/9/test_a.txt", TestLogger::createTestLogger()));
    }

    public function testResult(): void
    {
        $this->assertSame(1974232246, Solution9A::getResult("../resources/9/input.txt", TestLogger::createTestLogger()));
    }

}